pkg/platform/platform_test.go

```diff
@@ -45,3 +46,19 @@ func TestAppendVersionTag(t *testing.T) {
 		t.Fatalf("should append version tag to existing map")
 	}
 }
+
+func TestCommonControlPlaneChartsOrder(t *testing.T) {
+	expectedOrder := []string{"pod-checkpointer", "kube-apiserver", "kubernetes", "calico", "lokomotive", "bootstrap-secrets"} //nolint:lll
```

Comment:
Topic for other issue probably, but most likely `bootstrap-secrets` should be applied even before `pod-checkpointer`, so nodes can join the cluster for `pod-checkpointer` to converge.

Reply:
If we check if the worker nodes are up before applying `bootstrap-secrets` it will still fail.

Reply:
I know, but that's a separate issue :)